core/src/html/layout.rs

```diff
@@ -278,8 +279,7 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
         } else {
             self.font_leading_adjustment()
         };
-
-        if self.current_line_span.bullet && self.is_first_line {
+        if self.current_line_span.bullet && self.is_first_line && empty_text_line.unwrap_or(false) {
```

Comment:
I wonder if instead we can determine that the line is empty immediately here by looking at the `boxes` or `box_count`. Maybe:
`if self.current_line_span.bullet && self.is_first_line && box_count > 0`
Because if there are no boxes, then there is no text; but I'm not sure if this catches all edge cases. Will need testing to see if whitespace affects whether a bullet exists or not, etc.

Reply:
Testing a change which just added this line and the changes from https://github.com/ruffle-rs/ruffle/pull/5758, I found that this emulated the bullets on https://dnalc.cshl.edu/view/2022-G2C-3-D-Brain.html perfectly.